ares/md/apu/bus.cpp

```diff
@@ -89,7 +89,7 @@ auto APU::writeExternal(n24 address, n8 data) -> void {
 
 auto APU::in(n16 address) -> n8 {
   //unused on Mega Drive
-  return 0x00;
+  return 0xFF;
```

Comment:
Minor nitpick: it is code convention in ares to use lower case for constants like this

Reply:
fixed